1-exercises/J-string-substring/exercise3.js

```diff
@@ -8,7 +8,7 @@
 
 let statement = "I do not like programming";
 
-let result = "";
+let result = statement.substring(0, 5).concat(statement.substring(8, statement.length));
```

Comment:
A very very tiny mistake! you have one extra space between do and not! can you find out why?